docs/minikube.md

```diff
@@ -0,0 +1,383 @@
+# Minikube installation
+
+
+For the purporse of testing Insprd locally we recommmend using [minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+Following this tutorial you will see the necessary steps to have one of the Inspr's examples running locally, more specifically the `ping pong` demonstration, but it will work for any other.
+
+The requirements for this turorial are:
+- Docker Hub account
+- Docker installed
+- kubectl installed
+- Helm installed
+- insprctl installed
+- Skaffold installed
```

Comment:
Make this links to each of the installation methods. In my previous review comment I put all the links there